test/e2e/e2e-test.mk

```
@@ -0,0 +1,75 @@
+# Copyright 2022 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+E2E_TEST_OUTPUT ?= e2e-test-output.txt
+E2E_TEST_TIMEOUT ?= 60m
+GOTEST_VERBOSE ?= -v
+
+ifndef TANZU_API_TOKEN
+TANZU_API_TOKEN = ""
+endif
+
+ifndef TANZU_CLI_TMC_UNSTABLE_URL
+TANZU_CLI_TMC_UNSTABLE_URL = ""
+endif
+
+ifndef TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL
+TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL = gcr.io/eminent-nation-87317/tanzu-cli/test/v1/plugins/plugin-inventory:latest
+endif
+
+ifndef TANZU_CLI_E2E_TEST_LOCAL_CENTRAL_REPO_URL
+TANZU_CLI_E2E_TEST_LOCAL_CENTRAL_REPO_URL = localhost:9876/tanzu-cli/plugins/central:small
+endif
+
+
+.PHONY: e2e-cli-core ## Execute all CLI Core E2E Tests
+e2e-cli-core: e2e-cli-lifecycle e2e-cli-config e2e-cli-plugin-compatibility-test e2e-context-k8s-tests e2e-context-tmc-test e2e-cli-plugin-lifecycle-sync-test
```

For only this external facing test target, the help text should be printed with the target so it is shown with `make help` output.

```suggestion
e2e-cli-core: e2e-cli-lifecycle e2e-cli-config e2e-cli-plugin-compatibility-test e2e-context-k8s-tests e2e-context-tmc-test e2e-cli-plugin-lifecycle-sync-test ## Execute all CLI Core E2E Tests
```